File: Arrays/count_commas_range_I/TS/countCommas.ts
export {}

function countCommas(n: number): number{
    return Math.max(0, n - 1000 + 1)
}

const testCases: number[] = [
    1,
    999,
    1000,
    1002,
    10000,
    100000,
];

for (const n of testCases) {
    console.log(`Input:       ${n}`);
    console.log(`Output:      ${countCommas(n)}`);
    console.log("-".repeat(35));
}

File: Arrays/count_commas_range_I/go/count_commas.go
package main

import (
	"fmt"
	"strings"
)

func countCommas(n int) int {
	return max(0, n-1000+1)
}

func main() {
	testCases := []int{1, 999, 1000, 1002, 10000, 100000}

	for _, n := range testCases {
		fmt.Printf("Input:       %d\n", n)
		fmt.Printf("Output:      %d\n", countCommas(n))
		fmt.Println(strings.Repeat("-", 35))
	}
}
